tests/jerry/date-tostring.js

```diff
@@ -13,6 +13,47 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-assert (new Date(NaN) == "Invalid Date");
-assert (new Date("2015-02-13") == "2015-02-13T00:00:00.000");
-assert (new Date("2015-07-08T11:29:05.023") == "2015-07-08T11:29:05.023");
+assert (new Date (NaN) == "Invalid Date");
```

Comment:
Because it also checks the type, so it is more strict than the two. I would recommend to use it in the future.

Later you can make a style patch to change these.